platform/darwin/src/MGLCircleStyleLayer.h

```diff
@@ -43,6 +43,21 @@ typedef NS_ENUM(NSUInteger, MGLCircleTranslateAnchor) {
  instantiate a new circle layer to add to a map style or you may query an 
  `MGLMapView` for its `style` and obtain existing layers using the 
  `-[MGLStyle layerWithIdentifier:]` method. 
+
+ ### Example ###
+
+ ```swift
```

Comment:
These style layer headers are generated by platform/darwin/scripts/generate-style-code.js, so the next time someone runs `make style-code-darwin`, these changes will get blown away. The approach I took in #7475 was to override a property on each layer’s entry in the style specification JSON to provide documentation. Unfortunately, it’d be quite inconvenient to embed a multiline code example inside a JSON string.

The approach I’d recommend would be to have the overridden style specification JSON name the relevant function within MGLDocumentationExampleTests; generate-style-code.js can then extract that function using a regular expression and insert it into the generated header.